odoo/tools/_monkeypatches.py

```diff
@@ -11,6 +11,10 @@
 from werkzeug.wrappers import Request, Response
 
 from .json import scriptsafe
+import num2words
```

Comment:
You might want to wrap the imports in a try-except, like what is done for the other patches, so that people can run odoo without having to install the package